modules/highgui/src/plugin_wrapper.impl.hpp

```diff
@@ -235,12 +235,9 @@ std::vector<FileSystemPath_t> getPluginCandidates(const std::string& baseName)
     return results;
 }
 
-// NB: require loading of imgcodecs module
-static void* g_imwrite = (void*)imwrite;
 
 void PluginUIBackendFactory::loadPlugin()
 {
-    CV_Assert(g_imwrite);
```

Comment:
Please keep under `#ifdef` instead.

BTW, Removal/disabling of this code requires `OPENCV_PLUGIN_NO_LINK=OFF` to force plugin linking to opencv binaries.